pkg/chart/v2/util/values.go

```diff
@@ -183,6 +183,20 @@ func istable(v interface{}) bool {
 	return ok
 }
 
+func isNonEmptyTable(val interface{}) bool {
+	table, ok := val.(map[string]interface{})
+	if !ok {
+		return false
+	}
+
+	return len(table) > 0
+}
+
+func isNonEmptyString(val interface{}) bool {
+	valueString := fmt.Sprintf("%s", val)
```

Comment:
```suggestion
	valueString := fmt.Sprintf("%v", val)
```
I missed earlier discussion on this PR, seems like the format is wrong though.

Reply:
I included this change. thanks for spotting!